NorthstarDLL/util/printcommands.cpp

```diff
@@ -2,6 +2,16 @@
 #include "core/convar/convar.h"
 #include "core/convar/concommand.h"
 
+std::vector<std::pair<std::string, ConCommandBase*>> ConvarSort(std::unordered_map<std::string, ConCommandBase*> map)
```

Comment:
`std::vector<std::pair<std::string, ConCommandBase*>>` could be replaced with a shorter type like `ConCommandBases`. I just think these type names are too long.
 
 ```suggestion
typedef std::vector<std::pair<std::string, ConCommandBase*>> ConCommandBases;
 
ConCommandBases ConvarSort(std::unordered_map<std::string, ConCommandBase*> map)
 ```

Reply:
That just makes it less readable. If you want to go for readability the best course of action would be to get rid of `CCvar::DumpToMap` and use the internal iterator instead.

Another thing is imo ( gecko also mentioned this somewhere ) is we should strive to mimic engine structure part of that being using engine provided interfaces as much as possible. Making wrappers around wrappers just worsens code quality and maintainability.

Reply:
If it's preferred, I can rework all the convars to use the internal iterator instead of using `CCvar::DumpToMap`.
I had only used `CCvar::DumpToMap` as it matched the previous bob code™ which was already in the file.

Reply:
Switched to Internal Iterator in latest commit